lib/plugins/features/movies/controller.js

```diff
@@ -8,3 +8,14 @@ exports.create = (payload) => {
     return new Movie({ id: movie.id }).fetch();
   });
 };
+
+exports.list = (filter) => {
+  return new Movie().query((qb) => {
+    if (filter.title) {
+      qb.whereRaw('lower(title) = ?', filter.title.toLowerCase());
```

Comment:
The `.lowercase()` of [this line](https://github.com/kylekwong/sfmovies/pull/4/files#diff-72882ff564dfc03098e3b5fbb33ac550R6) already lowercases `filter.title` so `.toLowerCase()` is not necessary here.